aten/src/ATen/Dispatch.h

```diff
@@ -346,6 +361,25 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
     }                                                                       \
   }()
 
+#define AT_DISPATCH_QINT_AND_SUB_BYTE_TYPES(TYPE, NAME, ...)                                   \
```

Comment:
can we merge this with AT_DISPATCH_QINT_TYPES?

Reply:
Not currently. The AT_DISPATCH_QINT_TYPES is used for other kernels as well and if we try to dispatch over quint4x2, there will be errors in vec256 not being able to understand the scalar type